Straight_skeleton_2/examples/Straight_skeleton_2/CMakeLists.txt

```diff
@@ -9,10 +9,6 @@ find_package(CGAL REQUIRED COMPONENTS Core)
 
 find_package(CGAL COMPONENTS Qt5)
 
-if(CGAL_Qt5_FOUND)
-  add_definitions(-DCGAL_USE_BASIC_VIEWER -DQT_NO_KEYWORDS)
-endif()
-
 include(${CGAL_USE_FILE})
```

Comment:
Remove include(${CGAL_USE_FILE}) 
(I though we already removed all these use file ?)

Reply:
> Remove include(${CGAL_USE_FILE}) (I though we already removed all these use file ?)

That line is useless, but we have not yet made a pass over all `CMakeLIists.txt` to modernize them.